x-pack/test/observability_solution_api_integration/config/ess/config.base.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrConfigProviderContext } from '@kbn/test';
+import { services } from '../../../api_integration/services';
+
+export interface CreateTestConfigOptions {
+  testFiles: string[];
+  junit: { reportName: string };
+}
+
+export function createTestConfig(options: CreateTestConfigOptions) {
+  return async ({ readConfigFile }: FtrConfigProviderContext) => {
+    const xPackApiIntegrationTestsConfig = await readConfigFile(
+      require.resolve('../../../api_integration/config.ts')
+    );
+
+    const svlSharedConfig = await readConfigFile(
+      require.resolve('../../../../test_serverless/shared/config.base.ts')
```

Comment:
I import shared services from test_serverless, since I want to reuse the `sloApi` and `alertingApi`

Reply:
@pheyos As per our discussion I moved these 2 services under deployment agnostic services in [ this commit](https://github.com/elastic/kibana/pull/183113/commits/4d2182099a566d06c99cfd7085ef0ca292efc6f2?show-viewed-files=true&file-filters%5B%5D=) and tests still work fine! Does it look better this way?